src/DataService/Annotation/DataService.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Lullabot\Mpx\DataService\Annotation;
+
+/**
+ * @Annotation
+ *
+ * Each MPX data service exposes one or more objects. We require annotations
```

Comment:
I'm not a huge fan of 'DataService', but it's what MPX calls each service in their docs, and `MpxDataService` seemed redundant with the namespace.